public/404.html

```diff
@@ -58,10 +62,14 @@
   <!-- This file lives in public/404.html -->
   <div class="dialog">
     <div>
-      <h1>The page you were looking for doesn't exist.</h1>
+      <h2>Publify</h2>
+      <h1>404 Not Found</h1>
```

Comment:
I think the 404 message is the main thing, so it makes sense for that to be h1. I would remove the Publify heading entirely.